includes/update_handles.batch.inc

```diff
@@ -55,6 +55,7 @@ function islandora_handle_update_handles_batch_operation(array $params, &$contex
       )));
     }
     module_load_include('inc', 'islandora_handle', 'includes/handle');
+    module_load_include('inc', 'islandora', 'includes/derivatives');
```

Comment:
Should... these includes actually be outside of this `if`/`else` business?... If this was coming in with things serialized in the sandbox, this wouldn't actually reload these files... would it?

Reply:
Actually... this would probably go kaboom if things were serialized, with the file handle above... I... am spooked.